requirements/functests.in

```diff
@@ -0,0 +1,4 @@
+-r tests.txt
```

Comment:
```suggestion
-r requirements.txt
```

We don't generally want all unit test requirements to also be functest requirements (this is how the other projects do it)

Reply:
Done